### PR TITLE
feat: 2.0.0 EmptyState component

### PR DESCRIPTION
**Pull request by athill**

### CODE REVIEW

File: src/components/EmptyState/EmptyState.md
Comment (burnumd) [inline src/components/EmptyState/EmptyState.md:25]: This line should be indented

**Review by burnumd — CHANGES_REQUESTED** [review]

File: src/components/EmptyState/EmptyState.md
Comment (athill) [reply src/components/EmptyState/EmptyState.md:13]: Fixed. Thanks!

File: src/components/EmptyState/EmptyState.md
Comment (athill) [reply src/components/EmptyState/EmptyState.md]: Fixed. Thanks!

File: src/components/EmptyState/EmptyState.jsx
Comment (ri-pandey) [inline src/components/EmptyState/EmptyState.jsx:25]: Can this be replaced by `{subComponents}`, instead of the map?

**Comment by ri-pandey** [comment]
@athill I noticed that our approach of separating child components into their own modules only works if the components aren't wrapped in Rivet.rivetize. I wanted to give you a heads up while I investigate this.

**Comment by ri-pandey** [comment]
@athill I managed to fix the root issue on my branch feature/2-0-0-header. You can either rebase with my branch, or wait until I merge my Header branch. Although that won't be for until next week.

**Comment by athill** [comment]
@ri-pandey Let me know when you merge yours, please.

**Review by johglove — APPROVED** [review]

File: src/components/EmptyState/EmptyState.md
Comment (basham) [inline src/components/EmptyState/EmptyState.md:42]: The `svg` should be indented.

File: src/components/EmptyState/EmptyState.jsx
Comment (basham) [inline src/components/EmptyState/EmptyState.jsx:16]: Restricting the children to only the Actions or Content components seems like an unnecessary restriction, which may cause frustration if the user needs to do something a little different — like even including an extra `div` or something. It should be on the user to use the components in the right way. I recommend to just let the children pass through untouched.

File: src/components/EmptyState/EmptyState.jsx
Comment (basham) [inline src/components/EmptyState/EmptyState.jsx:11]: Remove this icon. It had to do with the approach that was just replaced.

File: src/components/util/RivetIcons.jsx
Comment (basham) [inline src/components/util/RivetIcons.jsx:41]: I think this is a typo. It should be `iconBell`, not `icoBell`.

File: src/components/EmptyState/Actions.jsx
Comment (basham) [inline src/components/EmptyState/Actions.jsx:6]: Should `className` merge (rather than replace) classes brought in with `attrs.className`? Is this the pattern done with other components?

File: src/components/EmptyState/Content.jsx
Comment (basham) [inline src/components/EmptyState/Content.jsx:6]: Should `className` merge (rather than replace) classes brought in with `attrs.className`? Is this the pattern done with other components?

**Review by basham — CHANGES_REQUESTED** [review]

File: src/components/util/RivetIcons.jsx
Comment (burnumd) [reply src/components/util/RivetIcons.jsx:41]: Yeah, I agree. The rest of the icons are labeled as `ico*`, though. I think we can change them in another PR.

File: src/components/EmptyState/Actions.jsx
Comment (burnumd) [reply src/components/EmptyState/Actions.jsx:6]: Good question, I'll need to look how we're handling that.

**Review by basham — APPROVED** [review]

**Pull request merged**
